### ISSUE TITLE
[BUG] Dont expose certificate password in stdout

### ISSUE THREAD

**Issue by khainke** [YehudaKremer/msix#87]
### :information_source: Info
Version: `2.8.7`

### :speech_balloon: Description
When running the `flutter pub run msix:create` the certificate password will be exposed on stdout:

```
[ +167 ms] executing: C:\Users\****\flutter\bin\cache\dart-sdk\bin\dart __deprecated_pub run msix:create -c
C:\Users\path\to\my\certificate\cert.pfx -p myCertPassword -v 0.0.1.0
```

I want to hand over the command line arguments for the path and password for the certificate via environment variables (`$CertPath`, `$CertPassword`) to not expose them, but they are still exposed on the stdout when running the `create` command:

```
flutter pub run msix:create -c $CertPath -p $CertPassword -v 0.0.1.0
[  +96 ms] executing: [C:\Users\****\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +525 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +2 ms] 5f105a6ca7a5ac7b8bc9b241f4c2d86f4188cf5c
[   +1 ms] executing: [C:\Users\****\flutter/] git tag --points-at ****
[  +52 ms] Exit code 0 from: git tag --points-at ****
[        ] 2.10.0
[  +12 ms] executing: [C:\Users\****\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +33 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[   +1 ms] origin/stable
[        ] executing: [C:\Users\****\flutter/] git ls-remote --get-url origin
[  +26 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms] https://github.com/flutter/flutter.git
[ +145 ms] executing: [C:\Users\****\flutter/] git rev-parse --abbrev-ref HEAD
[  +43 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +2 ms] stable
[ +116 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +8 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +6 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +120 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[        ] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[ +167 ms] executing: C:\Users\****\flutter\bin\cache\dart-sdk\bin\dart __deprecated_pub run msix:create -c
C:\Users\path\to\my\certificate\cert.pfx -p myCertPassword -v 0.0.1.0
✓ parsing cli arguments
✓ validating config values
✓ cleaning temporary files
✓ creating app icons folder
✓ generating icons
✓ copying app icons
✓ copying VC libraries
✓ getting certificate publisher
✓ generate appx manifest
✓ generate package resource indexing files
✓ packing
✓ cleaning temporary files
✓ signing
❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚❚ 100%
Msix Installer Created:
C:\Users\****\build\windows\runner\Release\my_app.msix
[+16400 ms] "flutter run" took 16.750ms.
[ +114 ms] ensureAnalyticsSent: 103ms
[   +4 ms] Running shutdown hooks
[        ] Shutdown hooks complete
[        ] exiting with code 0

```

### :scroll: Pubspec.yaml
```
dev_dependencies:
  msix: ^2.8.7
  
 # ...
 
msix_config:
  display_name: my.display.name
  publisher_display_name: my publisher
  identity_name: come.example.myapp
  logo_path: some-path/some-logo.png
  capabilities: "internetClientServer,broadFileSystemAccess,location"
  store: false
  dont_install_cert: true
```

**Comment by YehudaKremer** [comment]
hello

if you replace the `-v 0.0.1.0` part with `--version 0.0.1.0` it solve it?

**Comment by khainke** [comment]
Yes, it does. So does the `-v` also trigger a verbose mode somewhere down the line?

**Comment by YehudaKremer** [comment]
Yes, i fix this in 3.0v

Thank you for reporting this problem

בתאריך יום ב׳, 7 בפבר׳ 2022, 01:12, מאת khainke ***@***.***>:
